### DISCUSSION TITLE
RFC: Support a folder structure for multiple cluster

### DISCUSSION THREAD

**Discussion by onedr0p** [onedr0p/cluster-template#1238] [Ideas]
I have done this over at my home-ops repo and would love to carry those changes over here.

**Comment by mitchross** [comment]
upvote

**Comment by oscaromeu** [comment]
+1

**Comment by onedr0p** [comment]
I was planning on doing it like this where each cluster can pull from a global config/helm repos and also have it's own.

https://github.com/kashalls/home-cluster/tree/70ae28b2afa2638941a1184f77ffe901a30125b5/kubernetes

**Comment by mitchross** [comment]
> I was planning on doing it like this where each cluster can pull from a global config/helm repos and also have it's own.
> 
> https://github.com/kashalls/home-cluster/tree/70ae28b2afa2638941a1184f77ffe901a30125b5/kubernetes

Where does the server/vm/cluster IP's live in this? And how does it know where to deploy for each cluster.

**Comment by onedr0p** [comment]
Each cluster will have it's own Flux vars as well as a "global" vars to pull from.

**Comment by amasolov** [comment]
+1 for sure

**Comment by Vikaspogu** [comment]
+1

**Comment by kinzokudev** [comment]
+100. Would love to see this implemented
